### PR TITLE
mark_selection_unread

### PR DESCRIPTION
**Pull request by Twilek-de**
Translated that

### CODE REVIEW

**Comment by Alkarex** [comment]
Thanks for all the translations 👍 
Another time, please make the edits in the /dev branch instead of /master (I was able to retarget the correct branch without problem, but it does not always work in case of version conflicts)

File: app/i18n/de/index.php [hunk 1]
```diff
@@ -40,7 +40,7 @@
 		'mark_all_read' => 'Alle als gelesen markieren',
 		'mark_cat_read' => 'Kategorie als gelesen markieren',
 		'mark_feed_read' => 'Feed als gelesen markieren',
-		'mark_selection_unread' => 'Mark selection as unread',	//TODO
+		'mark_selection_unread' => 'Auswahl als gelesen markieren',
```
Comment (Frenzie) [inline app/i18n/de/index.php]: I think you missed an `un(gelesen)`. ;-)

Comment (Twilek-de) [reply app/i18n/de/index.php]: Ah chute I missed that
```suggestion
		'mark_selection_unread' => 'Auswahl als ungelesen markieren',
```

Comment (Frenzie) [reply app/i18n/de/index.php]: You'll have to commit the change yourself. I don't have write access to your repo. :-)

Comment (Twilek-de) [reply app/i18n/de/index.php]: A thank you, I am new to the whole committing thing :-) I hope that did it...

**Comment by Frenzie** [comment]
Thanks!

**Pull request merged**